media-libs/tg_owt/tg_owt-0_pre20210309.ebuild

```diff
@@ -0,0 +1,76 @@
+# Copyright 2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake
+
+TG_OWT_COMMIT="7f965710b93c4dadd7e6f1ac739e708694df7929"
+LIBVPX_COMMIT="5b63f0f821e94f8072eb483014cfc33b05978bb9"
+LIBYUV_COMMIT="ad890067f661dc747a975bc55ba3767fe30d4452"
```

Comment:
bundling is really bad and one of the reasons I've stepped down as maintainer.

I see you've started unbundling work in https://github.com/desktop-app/tg_owt/pull/55
We can make an exception this time, because tg needs a bump pretty urgently, but I really hope your patch gets merged and we could use system versions of libuv, libvpx and openh264.

Reply:
I'm less concerned about other libs, like abseil, video/media stuff is a priority due to CPU flag situation and lack of handling it upstream.

Reply:
I've been working on this for a week and I just want to upgrade telegram at this point. This is *known* to work and it gives me time to work out the kinks with the unbundling PR.
That said, libyuv has no stable versions and probably will never end up in the tree for that reason. Similarly, libvpx is currently using a git release.
I get why bundling is bad, but I can't do much about it in this case, so if it works, it works...